charts/datadog/templates/NOTES.txt

```diff
@@ -209,6 +209,30 @@ More information about this change: https://github.com/DataDog/helm-charts/pull/
 {{- fail "You are using datadog.systemProbe.enabled which has been superseded by networkMonitoring.enabled, systemProbe.enableTCPQueueLength, systemProbe.enableOOMKill, and securityAgent.runtime.enabled. These options provide a more granular control of which features should be activated." }}
 {{- end }}
 
+{{- if eq (include "should-enable-system-probe" .) "true" }}
+  {{- $healthPort := .Values.agents.containers.systemProbe.healthPort }}
+  {{- with $liveness := .Values.agents.containers.systemProbe.livenessProbe.httpGet }}
```

Comment:
Where is `.Values.agents.containers.systemProbe.livenessProbe.httpGet` specified?

Reply:
Interesting. I followed a similar pattern to how the core agent checks for misconfiguration here: https://github.com/DataDog/helm-charts/blob/main/charts/datadog/templates/NOTES.txt#L29-L49, but I can't seem to find where `.Values.agents.containers.agent.livenessProbe.httpGet` is specified either.

Maybe I am missing something, but if not, I'll add the appropriate fields to the `.Values.agents.containers.systemProbe.livenessProbe` object.